deployment/aws/installer/entry_point_template.yml

```diff
@@ -32,8 +32,7 @@
     JAVA_OPTS: |
       -Dcontroller.service.rpc.listener.port=9090
       -Dcontroller.service.rest.listener.port=10080
-      -Dcontroller.zk.url=ZKNODE:2181
-      -Dconfig.controller.metricenableCSVReporter=false
+      -Dcontroller.zk.connect.uri=ZKNODE:2181
```

Comment:
Thanks, the CSV Reported was removed as an option since the introduction of micrometer, but it looks like there were still leftovers from that work.